src/components/NumberField/NumberField.stories.tsx

```diff
@@ -27,6 +27,21 @@ export default {
   },
 };
 
+export const NumberFieldSizes = {
```

Comment:
Add a section for this also in the mdx file because otherwise it will not be visible in storybook

Reply:
Fixed with https://github.com/Orfium/orfium-ictinus/pull/849/commits/a71a5ee04262e2726f7b858a5197399b610478e0